pkg/autoscaler/metrics/http_scrape_client.go

```diff
@@ -20,12 +20,17 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"net/http"
 
 	dto "github.com/prometheus/client_model/go"
 	"github.com/prometheus/common/expfmt"
 )
 
+const (
+	protoAcceptContent = "application/protobuf"
```

Comment:
can't we use the same constant?
We can put it in `pkg/network` or thereabouts.

Reply:
Ok makes sense I didnt want to put it in an arbitrary pkg.